weed/shell/command_volume_fsck.go

```diff
@@ -72,6 +75,7 @@ func (c *commandVolumeFsck) Do(args []string, commandEnv *CommandEnv, writer io.
 	c.forcePurging = fsckCommand.Bool("forcePurging", false, "delete missing data from volumes in one replica used together with applyPurging")
 	purgeAbsent := fsckCommand.Bool("reallyDeleteFilerEntries", false, "<expert only!> delete missing file entries from filer if the corresponding volume is missing for any reason, please ensure all still existing/expected volumes are connected! used together with findMissingChunksInFiler")
 	tempPath := fsckCommand.String("tempPath", path.Join(os.TempDir()), "path for temporary idx files")
+	cutoffTimeAgo := fsckCommand.Uint64("cutoffTimeAgo", uint64(time.Now().Add(-time.Minute*30).Unix()), "the offset of filtering files on volume server")
```

Comment:
similar to this https://github.com/seaweedfs/seaweedfs/blob/358ccb5e0eb2fd0d5f7f1bd6259dc5950df76526/weed/command/filer_backup.go#L35

Reply:
fixed